.github/workflows/ci.yml

```diff
@@ -25,7 +43,9 @@ jobs:
           toolchain: ${{ matrix.rust }}
           override: true
       - name: Run cargo test
-        run: cargo test
+        # Exclude doctests here, as we don't want to clutter docs themselves
+        # with backwards compatibility workarounds.
+        run: cargo test --lib
```

Comment:
Some of the versions in the list above should still work here, would it be possible to still run the full test suite for them?

Also, stable/beta/nightly now are tested twice for no good reason.